apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/web.tsx

```diff
@@ -10,7 +10,10 @@ interface ICategory {
   [component: string]: CategoryPage;
 }
 
-const categories: { Other?: ICategory; [name: string]: ICategory } = {
+// Exporting this object to be used in generating a TOC (table of content) for docs.microsoft documentation repo.
+// Any changes to this object need to be communicated to `vibraga@microsoft.com` or any member of Gearbox team
```

Comment:
Hrm this seems fragile. Could we produce an intermediary JSON file artifact or similar as part of a build and consume it for docs.microsoft.com? Perhaps we can run validation against that artifact as well to ensure it is not broken / matches some schema.

Either that or ensure that the relevant folks are codeowners of this file to enforce the above comment.

Reply:
`api-documenter` action we are going to use the config file generated after parsing this object and building a sort of schema, will [load and validate](https://github.com/microsoft/web-build-tools/pull/1265/files#diff-5d5acd0ae6cd3e7f13d6253cdfd2379bR29) it. At the same time all I need is this object and don't know if that makes it more difficult to extract and produce an artifact.

Adding myself as a code-owner could also be something to look into.

Reply:
CODEOWNER seems reasonable for now; you may no longer need the explicit alias callout via doc comment.

Longer term some sort of validation against this object and perhaps consuming the JSON via a build artifact is a more robust approach, but we'd need to design it a tad.